templates/home/index.html.twig

```diff
@@ -1,7 +1,87 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}{{ title }}{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
-{% endblock %}
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% if studentArray %}
+            {% for key in studentArray|keys %}
```

Comment:
Twig palaiko `{% for key, value`. Tada nereikės rašyti dvigubo `for`

Plačiau:
 * https://twig.symfony.com/doc/2.x/tags/for.html#iterating-over-keys-and-values